src/components/Login/SignIn.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react'
+import { connect } from 'react-redux'
+import { useLogin } from 'react-admin'
+
+import AuthProvider from '../../providers/authProvider'
+
+const SignIn = props => {
+  function onSignIn() {
```

Comment:
Just for convetions, you should use const whereever you can use it.

Reply:
Ohh ok. Thanks for the feedback. I will